buildpack.md

```diff
@@ -42,14 +42,12 @@ This is accomplished in two phases:
 10. [Security Considerations](#security-considerations)
     1. [Assumptions of Trust](#assumptions-of-trust)
     2. [Requirements](#requirements)
-11. [Artifact Format](#artifact-format)
-    1. [Buildpack Package](#buildpack-package)
-12. [Data Format](#data-format)
-    1. [buildpack.toml (TOML)](#buildpack.toml-toml)
-    2. [launch.toml (TOML)](#launch.toml-toml)
-    4. [Build Plan (TOML)](#build-plan-toml)
-    5. [Bill-of-Materials (TOML)](#bill-of-materials-toml)
-    6. [Layer Content Metadata (TOML)](#layer-content-metadata-toml)
+11. [Data Format](#data-format)
+    1. [launch.toml (TOML)](#launch.toml-toml)
+    2. [Build Plan (TOML)](#build-plan-toml)
```

Comment:
there's a build plan and a buildpack plan?

Reply:
The input into `/bin/build` is just the `require`s (provided to the buildpacks that said they'd provide them), while the output of `/bin/detect` is both `require`s and `provide`s. Ben pointed out that the spec is currently incorrect and confusing because there's only one format mentioned. For now, I just renamed the stuff that goes into `/bin/build` "buildpack plan" as an improvement over the current version.

Reply:
@sclevine but the schema shown below looks quite different from what you described. I'm not really clear on what `[[entries]]` is used for.

Reply:
It's just the name of the array. The contents are the same as `requires` coming out of `/bin/detect`. I thought it would be the same format as the BOM, but the BOM also includes the buildpack list, so I split it out into a new format section.